go-controller/pkg/ovn/egressip.go

```diff
@@ -2258,6 +2274,28 @@ func (e *egressIPDial) dial(ip net.IP, timeout time.Duration) bool {
 	return true
 }
 
+func (oc *Controller) isReachableViaGRPC(node *egressNode, healthCheckPort int) bool {
+	// Check if we need to do node reachability check
+	if oc.eIPC.egressIPTotalTimeout == 0 {
+		return true
+	}
+
+	if len(node.mgmtIPs) == 0 {
+		return false
```

Comment:
how is this possible? Is this a race when the node is first added? Would it be wrong to consider it as a failure when the node hasnt initialized yet?

Reply:
heh, that is a copy and paste from the legacy implementation. I don't really expect that to ever happen. I will remove it from here.